setup.py

```diff
@@ -41,23 +48,20 @@ def find_version(*file_paths):
         "pyformlang",
         "protobuf",
         "fastapi",
-        "uvicorn"
+        "uvicorn",
     ],
     extras_require={
-        'docs': [
-            'ipython',
-            'numpydoc',
-            'sphinx_rtd_theme',
-            'sphinx',
-            'nbsphinx'
-        ],
-        'test': [
+        "docs": ["ipython", "numpydoc", "sphinx_rtd_theme", "sphinx", "nbsphinx"],
+        "test": [
+            "accelerate",
             "jsonschema",
-            'pytest',
-            'transformers',
-            'torch',
-            'pytest-cov'
-        ]
+            "jupyter",
+            "papermill",
+            "pytest",
+            "pytest-cov",
+            "transformers",
+            "torch",
+            "transformers",
```

Comment:
let's keep this and drop the other `black` format changes to setup.py for now (completely OK with these testing dependencies being added)